src/microbe_stage/Patch.cs

```diff
@@ -423,6 +442,21 @@ public override string ToString()
     {
         return $"Patch \"{Name}\"";
     }
+
+    private float GetSunlightAmount(CompoundAmountType option)
+    {
+        switch (option)
+        {
+            case CompoundAmountType.Maximum:
+                return Biome.Sunlight!.Maximum * 100;
```

Comment:
The 100 here and in `GetCompoundAmountInSnapshot` should be extracted as a constant as this is now used in a bunch of places as a magic value.

Reply:
Doesn't that 100 just convert it from a decimal percentage to a whole (for lack of a better term) percentage?

Reply:
Well the method names are not clear at all what they return. And also the "normal" compounds don't seem to be multiplied by anything. I'm definitely guilty of using 100 as a suitable value multiplier without intending it to be taken as a percentage value.

Reply:
At least a summary for the methods should be added explaining what kind of values it returns.